### PR TITLE
Add `remove_token_from_body_when_cookies_used` config option

### PR DESCRIPTION
**Pull request by TjorvenB**
Added configuration and documentation for the argument added in Lexik#912. https://github.com/lexik/LexikJWTAuthenticationBundle/pull/912

### CODE REVIEW

File: Resources/doc/1-configuration-reference.md [hunk 1]
```diff
@@ -70,6 +70,9 @@ lexik_jwt_authentication:
             cookies:
                 - jwt_hp
                 - jwt_s
+                
+    # do not remove the token from the body when using cookies
```
Comment (chalasr) [inline Resources/doc/1-configuration-reference.md]:
```suggestion
    # remove the token from the response body when using cookies
```

**Comment by TjorvenB** [comment]
Hello, is there something that is blocking this PR? If so, please let me know and I will make the required modifications.

**Review by chalasr — APPROVED** [review]
Looks good. One minor comment

**Comment by chalasr** [comment]
This looks good. I'm going to merge it a bit later this month. Thank you!

**Comment by lwillems** [comment]
thx @TjorvenB and @chalasr needs it too ;)

**Comment by chalasr** [comment]
Thank you @TjorvenB.

**Pull request merged**